contrib/scripts/checkpr.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+if [ $# -eq 0  ]
+then
+	echo "Usage: checkpr.sh PULL_REQUEST_ID"
+	exit
+fi
+
+PR_ID=$1
+
+result="null"
+
+until [ "$result" != "null" ]; do
+	sleep 60
+	result=$(curl -s https://jenkins.cilium.io/job/cilium/job/cilium/job/PR-${PR_ID}/lastBuild/api/json | jq ".result")
+done
+
+name=$(uname)
+for run in {1..5}
+do
+	if [ "$name" == "Linux" ]; then
+		paplay /usr/share/sounds/gnome/default/alerts/bark.ogg
```

Comment:
Would you mind wrapping this with a `if [ -e $BARK_PATH ]; then` so the alternative will fall back to something that should be more widely available, perhaps like `paplay /usr/share/sounds/freedesktop/stereo/bell.oga`?